tests/automation_framework/data/work_order/workordersubmit_requesterSignature_noencryptedRequestHash.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Similar to the above comment, why does the file name have requesterSignature? I don't see any changes to the RequesterSignature field.